### PR TITLE
Feature/budget summary information section iteration one

### PR DESCRIPTION
**Pull request by alejandrocabriales**
# Ticket

https://trello.com/c/JHKHOrzX/294-feature-budget-summary-navigation

# What solved
- [X] Create a Level Budget Summary Box  with all the filters(default Budget)

# Description
Create a Level Budget Summary Box  with all the filters(default Budget)

### CODE REVIEW

File: src/stories/containers/Finances/FinacesContainer.tsx [hunk 1]
```diff
@@ -1,28 +1,79 @@
...
 import MakerDAOExpenseMetrics from './SeccionPages/MakerDAOExpenseMetrics';
+import { useFinances } from './useFinances';
+import type { WithIsLight } from '@ses/core/utils/typesHelpers';
 
-const FinancesContainer = () => (
-  <PageContainer>
-    <Container>
-      <div>MakerDAO Finances </div>
+const FinancesContainer = () => {
+  const { isLight } = useThemeContext();
```
Comment (YasielCabrera) [inline src/stories/containers/Finances/FinacesContainer.tsx:16]: Could you move this to the hook in order to have only one "logic" operation here?

**Review by YasielCabrera — CHANGES_REQUESTED** [review]
RC

File: src/stories/containers/Finances/FinacesContainer.tsx (hunk 1 above)
Comment (alejandrocabriales) [reply src/stories/containers/Finances/FinacesContainer.tsx]: you are right donde

**Review by YasielCabrera — APPROVED** [review]
lgtm

**Pull request merged**